### PR TITLE
37 update capsules to aind data schema 11

### PR DESCRIPTION
**Pull request by arielleleon**

### CODE REVIEW

**Comment by arielleleon** [comment]
Moving to draft until I am ready to bump the version on all capsules

**Comment by seanmcculloch** [comment]
Updates included in this PR:

- Upgrading aind-data-schema 0.22.0 -> 1.1.0
- Change processing metadata strategy to serializing DataProcess object
- Creation and serialization of two QCMetrics
- Updates to multiplane_motion_correction()
  - look for h5 with "*VI*" from "ophys_experiment"
  - get frame_rate_hz from session.json, fallback to platform.json

File: code/registration.py [hunk 1]
```diff
@@ -269,6 +263,171 @@ def load_initial_frames(
     return frames
 
 
+def combine_images_with_individual_titles(image1_path, image2_path, output_path, title1="", title2=""):
```
Comment (arielleleon) [inline code/registration.py:266]: Can you add types to the inputs? E.g

`def combine_images_with_individual_titles(image1_path: str, image2_path: str, output_path: str, title1: str =1, title2: str=2`

File: code/registration.py [hunk 2]
```diff
@@ -269,6 +263,171 @@ def load_initial_frames(
...
+    """
+    Combine two images side-by-side with padding and add individual titles above each image.
+
+    Parameters:
```
Comment (arielleleon) [inline code/registration.py:270]: Can you change the docstrings to numpy docstrings format?

**Comment by arielleleon** [comment]
@seanmcculloch: looks good! just a few stylistic adds.

File: code/registration.py (hunk 1 above)
Comment (seanmcculloch) [reply code/registration.py]: Added parameter types

File: code/registration.py (hunk 2 above)
Comment (seanmcculloch) [reply code/registration.py]: Docstrings updated

**Review by arielleleon — COMMENTED** [review]
Approve!

**Pull request merged**